app/models/calculationList/CalculationListModel.scala

```diff
@@ -20,6 +20,10 @@ import play.api.libs.json.{Format, Json}
 
 sealed trait CalculationListResponseModel
 
+object CalculationListResponseModel {
```

Comment:
I am not sure if we really should do anything with response model here;
Is that to do with 1896 API?

Reply:
This is for parsing the response into Json for the test only page:

OptOutTestDataController:
` Ok(s"Crystallisation Status:    ${Json.toJson(seqResult._1)}\n"`